app/src/main/res/layout/item_hotel.xml

```diff
@@ -111,82 +112,99 @@
             android:layout_height="wrap_content"
             android:layout_marginStart="@dimen/common_margin_m3"
             android:layout_marginTop="2dp"
+            android:layout_marginEnd="@dimen/common_margin_m3"
             android:ellipsize="end"
             android:maxLines="1"
             android:textColor="@color/contentHeaderTextColor"
-            app:layout_constraintEnd_toStartOf="@id/hotel_price_from"
+            app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/hotel_name" />
+            app:layout_constraintTop_toBottomOf="@id/hotel_name"
+            tools:text="Distance to a landmark 3km offering rooms in a way you could never imagine " />
 
-        <TextView
-            android:id="@+id/hotel_rating"
-            android:layout_width="wrap_content"
-            android:layout_height="24dp"
-            android:layout_marginStart="@dimen/common_margin_m3"
-            android:layout_marginTop="10dp"
-            android:background="@drawable/bg_travel_card_info"
-            android:drawableStart="@drawable/ic_rating_blue"
-            android:drawablePadding="5dp"
-            android:fontFamily="sans-serif-medium"
-            android:gravity="center_vertical"
-            android:paddingStart="5dp"
-            android:paddingEnd="6dp"
-            android:textColor="#007aff"
-            android:textSize="@dimen/travel_hotel_card_info_text_size"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/hotel_description"
-            tools:text="7.5/10" />
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
I think it's not necessary to nest this ConstraintLayout. I'll merge it first, please help to refine the layout later.